tests/python-gpu/test_gpu_prediction.py

```diff
@@ -212,6 +212,9 @@ def test_shap(self, num_rounds, dataset, param):
            tm.dataset_strategy, shap_parameter_strategy)
     @settings(deadline=None, max_examples=20)
     def test_shap_interactions(self, num_rounds, dataset, param):
+        if dataset.name == 'sparse':
+            return
```

Comment:
Can we use `pytest.xfail` or `pytest.skip` here? This way, pytest will show that this test was skipped and the reason why.

Reply:
Done.